mmv1/third_party/terraform/services/composer/resource_composer_environment.go.erb

```diff
@@ -1078,6 +1078,7 @@ func ResourceComposerEnvironment() *schema.Resource {
 							Type:         schema.TypeString,
 							Required:     true,
 							ForceNew:     true,
+							DiffSuppressFunc: tpgresource.CompareResourceNames,
```

Comment:
Would you be able to add a dedicated function to deal with the gcs prefix to tpgresource? This would work but mostly by coincidence- because we strip after the last `/`.